portfolio/src/main/webapp/me.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
```

Comment:
Does this file exist somewhere? If not, could you please make it or remove this line?

Also, you should probably include javascript after the body, so that when it runs, the content of the body is already available. Otherwise, you'll have to add a document.load event handler or something.

Reply:
These files do exist. They should have been included with the initial push I made to my repo